src/main/java/Avocado.java

```diff
@@ -0,0 +1,67 @@
+import java.util.Scanner;
+public class Avocado {
+    public static void main(String[] args) {
```

Comment:
You could consider refactoring this main method as it is too long, and to adhere to this code quality [guideline](https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods)